frontend/features/sectionBlocks/FeatureBlock.tsx

```diff
@@ -0,0 +1,204 @@
+/* eslint-disable @next/next/no-img-element */
+import { Box, chakra, Container, Flex, Img, Link } from "@chakra-ui/react";
+import { STRAPI_URL } from "@config/env";
+import React from "react";
+import {
+  Block,
+  BlockComponentProps,
+  BlocksControls,
+  InlineImage,
+  InlineTextarea,
+} from "react-tinacms-inline";
+import { useCMS } from "tinacms";
+import { BlockTemplateData } from "./types";
+
+export type FeatureBlockData = BlockTemplateData<
+  "ComponentBlocksSingleFeature",
+  {
+    id: Nullable<string>;
+    image: Nullable<FeatureImage>;
+    title: Nullable<string>;
+    description: Nullable<string>;
+    serviceLink: Nullable<string>;
```

Comment:
I think this field name should be `featureLink`. But I'd like to simplify this further and rename it to `url`